format.sh

```diff
@@ -0,0 +1,5 @@
+#! /usr/bin/env bash
```

Comment:
Just run ./format.sh to format code in repo. If you run this before ./lint.sh, then the linting tests will pass.